Add back `REVOKE ROLE role FROM user` in syntax table and other syntax updates (#747)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/dev/ql/administration/security/role-management-syntax.asciidoc

----
CREATE OR REPLACE ROLE name [AS COPY OF name]
----

> | Create a new role, replacing an existing role if one exists with the same name.

Reviewer: A suggestion for rewording:
"Create a new role, or if a role with the same name exists, replace it."

Reviewer: Or even: Create or replace a role with the specified name.

Author: I updated this wording once from our code review, from `Recreate a role, or create a new role if no such role exists.` to `Create a new role, replacing an existing role if one exists with the same name.`

Think I prefer your first version
```suggestion
| Create a new role, or if a role with the same name exists, replace it.
```
